Original message by Sergio Durigan Junior:

  <https://sourceware.org/ml/gdb-patches/2013-09/msg00470.html>
  Message-ID: <m3ioy0qxs7.fsf@redhat.com>

====

commit 6bc305f56c7f9612c87f7ba33d338de64560a725
Author: Sergio Durigan Junior <sergiodj@redhat.com>
Date:   Mon Sep 16 21:07:14 2013 +0000

    The error message in the function was saying
    "You should provide one parameter..." while it should be saying "... one
    argument...".  Replaced.
    
    2013-09-16  Sergio Durigan Junior  <sergiodj@redhat.com>
    
    	* value.c (isvoid_internal_fn): Replace "parameter" with
    	"argument".

Index: gdb-7.2/gdb/value.c
===================================================================
--- gdb-7.2.orig/gdb/value.c
+++ gdb-7.2/gdb/value.c
@@ -2595,7 +2595,7 @@ isvoid_internal_fn (struct gdbarch *gdba
   int ret;
 
   if (argc != 1)
-    error (_("You must provide one parameter for $_isvoid."));
+    error (_("You must provide one argument for $_isvoid."));
 
   ret = TYPE_CODE (value_type (argv[0])) == TYPE_CODE_VOID;
 
